### PR TITLE
Release 0.1.9

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/petab-specification-changes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: PEtab specification changes
+about: Suggest a change to the PEtab specification (beyond fixes / clarifications)
+title: ''
+labels: file format
+assignees: ''
+
+---
+
+**Which problem would you like to address? Please describe.**
+A clear and concise description of which use case you want to address and if applicable, why the current specifications do not fulfill your requirements.
```
Comment (yannikschaelte) [inline .github/ISSUE_TEMPLATE/petab-specification-changes.md:11]:
```suggestion
A clear and concise description of which use case you want to address and, if applicable, why the current specifications do not fulfill your requirements.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
 
+### 0.1.9
+
+Library:
+
+* Allow URL as filenames for YAML files and SBML models (Closes #187) (#459)
+* Allow model time in observable formulas (#445)
+* Make float parsing from CSV round-trip (#444)
+* Validator: Error message for missing IDs, with line numbers. (#467)
+* Validator: detect duplicated observable IDs (#446)
```
Comment (yannikschaelte) [inline CHANGELOG.md:13]: lower/upper case inconsistent :nerd_face:

File: doc/compile_doc.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Build a pdflatex document from the documentation
+
+# format A3
+pandoc documentation_data_format.rst -o documentation_data_format.pdf -V geometry:margin=1.5in -V geometry:a3paper -V fontsize=10pt --toc --pdf-engine pdflatex
```
Comment (yannikschaelte) [inline doc/compile_doc.sh:6]:
```suggestion
pandoc documentation_data_format.rst \
    -o documentation_data_format.pdf \
    -V geometry:margin=1.5in -V geometry:a3paper -V fontsize=10pt \
    --toc \
    --pdf-engine pdflatex
```

File: doc/compile_doc.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+# format A2 landscape
+
+# pandoc documentation_data_format.rst -o documentation_data_format.pdf -V geometry:margin=.5in -V geometry:a4paper -V geometry:landscape -V fontsize=10pt --toc --pdf-engine pdflatex
```
Comment (yannikschaelte) [inline doc/compile_doc.sh:10]:
```suggestion
# pandoc documentation_data_format.rst \
#     -o documentation_data_format.pdf \
#     -V geometry:margin=.5in -V geometry:a4paper -V geometry:landscape -V fontsize=10pt \
#     --toc \
#     --pdf-engine pdflatex
```

**Review by yannikschaelte — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (dweindl) [reply CHANGELOG.md:13]: For consistency with the rest of the file... :innocent:

**Pull request merged**